.github/workflows/ci.yml

```
@@ -519,13 +519,7 @@ jobs:
           nogil: true
       - run: python3 -m sysconfig
       - run: python3 -m pip install --upgrade pip && pip install nox
-      - run: nox -s ffi-check
-      - name: Run default nox sessions that should pass
-        run: nox -s clippy docs rustfmt ruff
-      - name: Run PyO3 tests with free-threaded Python (can fail)
-        # TODO fix the test crashes so we can unset this
-        continue-on-error: true
-        run: nox -s test
+      - run: nox
```

I think dropping `ffi-check` might be too far? It's not in the default sessions.

```suggestion
      - run: nox -s ffi-check
      - run: nox
```